k8s/demo/common/divorce/div-cos.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       environment:
         IDAM_API_URL: "https://idam-api.demo.platform.hmcts.net"
         FEATURE_DN_REFUSAL: "true"
-        FEATURE_PAPER_UPDATE: true
+        FEATURE_PAPER_UPDATE: false
```

Comment:
Can you write a comment explaining why this is off in demo, @yarpo ?

Reply:
According to what I've got from Leszek (BSP team) we don't need to feature toggle it off.

We need to provide them a uuid of doc uploaded and only these docs will be printed.